### DISCUSSION TITLE
Carpet Duplication glitch - not working with GriefPrevention?

### DISCUSSION THREAD

**Discussion by learningtoolsmrdan** [GriefPrevention/GriefPrevention#1613] [Q&A]
Hello, don't ask me why but players on my server want to use the carpet duplication glitch as shown in this video: https://www.youtube.com/watch?v=4jLY4EA8Mnw

It is not working on the server and I am not sure why. Is there any thing I can adjust in the GriefPrevention config to allow this glitch?

**Comment by Jikoo** [comment]
GriefPrevention does not block glitches. If you're using Paper, you probably need to use their unsupported settings section to enable duplication bugs.
